src/ReachSets/DiscretePost/ConcreteDiscretePost.jl

```diff
@@ -144,3 +143,13 @@ function post(𝒫::ConcreteDiscretePost,
             target_loc_id, jumps)
     end
 end
+
+# --- assignments ---
+
+function apply_assignment(𝒫::ConcreteDiscretePost,
+                          assignment::MathematicalSystems.LinearMap,
+                          R⋂G::LazySet;
+                          kwargs...)
+    # TODO converting to HPolytope ?? handle automatically ??
+    return convert(HPolytope, linear_map(assignment.A, R⋂G))
```

Comment:
we can now do this one with `apply(assignment, R⋂G)` or `linear_map(ans)` if we want that.